extensions/kafka/src/test/java/com/hazelcast/jet/kafka/impl/KafkaTestSupport.java

```diff
@@ -179,6 +243,33 @@ public <K, V> KafkaConsumer<K, V> createConsumer(
         return consumer;
     }
 
+    public void createSchemaRegistry(SchemaRegistryConfig config) throws Exception {
+        SchemaRegistryRestApplication schemaRegistryApplication = new SchemaRegistryRestApplication(config);
+        schemaRegistryServer = schemaRegistryApplication.createServer();
+        schemaRegistryServer.start();
+        schemaRegistry = schemaRegistryApplication.schemaRegistry();
+    }
+
+    public void shutdownSchemaRegistry() throws Exception {
+        if (schemaRegistryServer != null) {
```

Comment:
don't we need to shutdown entire `SchemaRegistryRestApplication`?

Reply:
`schemaRegistryServer = schemaRegistryApplication.createServer()` registers `schemaRegistryApplication` as an application in `schemaRegistryServer`. When the server shuts down, all registered application are also shut down.